tests/test_browser.py

```diff
@@ -4652,10 +4652,25 @@ def test_base64_atob_fallback(self):
 
   # Tests that SINGLE_FILE works as intended in generated HTML (with and without Worker)
   def test_single_file_html(self):
-    self.btest('emscripten_main_loop_setimmediate.cpp', '1', args=['-s', 'SINGLE_FILE=1', '-s', 'WASM=1'], also_proxied=True)
+    self.btest('single_file_static_initializer.cpp', '1', args=['-s', 'SINGLE_FILE=1', '-s', 'WASM=1'], also_proxied=True)
     self.assertExists('test.html')
     self.assertNotExists('test.js')
     self.assertNotExists('test.worker.js')
+    self.assertNotExists('test.wasm')
+    self.assertNotExists('test.mem')
+
+  # Tests that SINGLE_FILE works as intended in generated HTML with MINIMAL_RUNTIME
+  def test_minimal_runtime_single_file_html(self):
```

Comment:
could this fit into `test_single_file` perhaps, that is, run minimal runtime as one of the modes there?

Reply:
That tests the `also_proxied=True` mode, which it something MINIMAL_RUNTIME does not support, so thought it simplest to have this separate.